### PR TITLE
Include rewind feature in play docs

### PR DESCRIPTION
**Pull request by huygur**
## Description

This PR includes Play's Rewind feature documentation & limitations.

This should be merged after Rewind is live. First step would be this PR to be merged: https://github.com/camunda/web-modeler/pull/7705

## When should this change go live?

- [x] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.4/components/modeler/web-modeler/play-your-process.md
Comment (christinaausley) [inline versioned_docs/version-8.4/components/modeler/web-modeler/play-your-process.md:50]: I think I'm just curious how to actually rewind the process without a screenshot -- do I click an icon? How do I select where I rewind this to?

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:52]: Here you say `task`, but elsewhere you say `element`, which may include events.

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:56]: In the Modeler, it's called a "Call activity"

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:57]: Elsewhere in the page it says "Timer events are executed as defined, with the option to skip the wait."
```suggestion
- Timer events that complete without being skipped
```

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:52]: For clarity
```suggestion
After completing part of your process, you can **rewind** to a previous task to test a different scenario. Play will start a new instance and replay your actions up to, but not including, the selected previous task.
```

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:60]: For brevity and clarity
```suggestion
If you completed an unsupported element before rewinding, you will rewind farther than expected.
```

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:62]: Recommending a concrete example
```suggestion
In addition, Play rewinds to an element, not to an element instance. For example, if you wanted to rewind your process to a sequential multi-instance service task which ran five times, it will rewind your process to the first instance of that service task.
```

**Review by HanselIdes — CHANGES_REQUESTED** [review]
This is really clear and covers exactly what it needs to. I added a few comments and suggestions for clarity and to explain the value of the feature.
@christinaausley, I'll leave it to you to review writing style

File: versioned_docs/version-8.4/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [reply versioned_docs/version-8.4/components/modeler/web-modeler/play-your-process.md:50]: Good point. The screenshot should include the rewind icon, and the docs should mention that there's a rewind button

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md:52]: I copied this explanation from old Play's docs to be honest 😅  it seems element is more appropriate then, I'll fix it

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md:60]: this looks great, thanks!

File: versioned_docs/version-8.4/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [reply versioned_docs/version-8.4/components/modeler/web-modeler/play-your-process.md:50]: screenshot added 👍

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (HanselIdes) [reply docs/components/modeler/web-modeler/play-your-process.md]: Got it 😁 
I didn't pay as much attention to those docs

**Comment by huygur** [comment]
hey @christinaausley , this PR can be merged as the changes are live now.
but please let us know what you think about @HanselIdes comment here https://github.com/camunda/camunda-docs/pull/3236#discussion_r1464819996 
and I can do the necessary changes if needed

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (christinaausley) [reply docs/components/modeler/web-modeler/play-your-process.md:47]: "ranked from slowest to fastest" -- what do we mean by this? I like the additions, but don't know if that part is necessary.

Also, I originally was wondering if this was a series of steps -- given it is three separate options, I would make make this a tabbed list rather than a numerical list.

Comment (HanselIdes) [reply docs/components/modeler/web-modeler/play-your-process.md:47]: @christinaausley I meant that there are different ways to replay your process, and rewind is the fastest
If it's not clear, it can be dropped. Overall, I'm trying to push people to use rewind

Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md:47]: I have applied this change with minor changes (I changed the title so it fits the others, and removed the last sentence in the Play a process section so its not repeated in this new title you suggested)

@christinaausley please check the latest version and feel free to merge if it looks good

Comment (christinaausley) [reply docs/components/modeler/web-modeler/play-your-process.md:47]: I would likely just write out as follows:

To replay a process, rewind to an earlier element by clicking on the **Rewind** button on a previously completed element.

:::note

Comment (HanselIdes) [reply docs/components/modeler/web-modeler/play-your-process.md:47]: Christina's suggestion makes a ton of sense to me.  Using the note shows that they're less preferred, but still optional

Comment (christinaausley) [reply docs/components/modeler/web-modeler/play-your-process.md:47]: @huygur Once this is incorporated, I can approve and merge 🥳

Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md:47]: I made the changes, hope I understood it correctly 😅

**Review by christinaausley — APPROVED** [review]

**Comment by akeller** [comment]
Based on the convo on this PR, let's merge it! 🎉

**Pull request merged**
